arbitrum-docs/node-running/running-a-node.md

```diff
@@ -1,4 +1,11 @@
-# Running a Node
+---
+title: "Quickstart: Run a node"
```

Comment:
This doesn't follow the conventions in our style & convention guide yet but it will soon.